regress/expected.output

```diff
@@ -10,21 +10,21 @@ Loading /tests/test-008.c -> Ok.
 Loading /tests/test-009.c -> Ok.
 Loading /tests/test-010.c -> Ok.
 Loading /tests/test-011.c -> *too many alarms in object
-Loading /tests/test-012.c -> tests/test-012.c: redefinition of variable 'i' line 7
-tests/test-012.c: Type mismatch ( int * vs int ) when initializing i line 7
+Loading /tests/test-012.c -> /tests/test-012.c: redefinition of variable 'i' line 7
+/tests/test-012.c: Type mismatch ( int * vs int ) when initializing i line 7
```

Comment:
A number of differences here are just the leading slash. I'm just running `make check` locally to run these, so not sure.